### DISCUSSION TITLE
CEC LG TV

### DISCUSSION THREAD

**Discussion by rijsab** [awawa-dev/HyperHDR#473] [Q&A]
Hi,

I have my Apple TV connected to a USB grapper, ny Pi 4 is connected to the same USB grapper and from this USB grapper connected to my LG TV CEC HDMI connecties. Woensdag I turn off the Apple TV, my TV stads on. Same if I turn on my Apple TV, the TV stads off. Is there a solution to have the CEC connectie working between Apple TV and LG TV?

My LG TV recognize the Apple TV input by the way and I am able to use the TV remote to controle the Apple TV. 

Thanks and really love this project.

**Comment by awawa-dev** [comment]
Hi
Your CEC is working because you are using your LG remote to control Apple TV. Probably some option must be enabled on the TV or Apple but I'm not familiar with them.

**Reply by rijsab** [reply]
Cheers, all settings in TV and ATV are correct. If I bypass the usb grabber and connect ATV directly to the CEC of TV, everything works as it should be and I can use the ATV remote to shutt off and on the TV.

**Reply by awawa-dev** [reply]
You said your TV remote worked with the Apple TV. Now it seems that your grabber is blocking CEC (completely, not even TV remote should work) which is a common side-effect of using USB grabbers with a loop.

**Reply by rijsab** [reply]
Correct, my LG remote works fine with my Apple TV. My TV recognize ATV as input as well, but still off or on is still nog working.

**Reply by awawa-dev** [reply]
USB grabbers with a loop block CEC

**Reply by rijsab** [reply]
Is there anything I can do to keep hyperhdr and have at the same time cec working?

**Reply by awawa-dev** [reply]
Some HDMI splitters can forward CEC on one port, this way you connect this port to the TV and the second port to the USB grabber.
